config/storage.yml

```diff
@@ -0,0 +1,35 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
+
+# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
+# amazon:
+#   service: S3
+#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
+#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
+#   region: us-east-1
+#   bucket: your_own_bucket
```

Comment:
I'm tempted to say the region and bucket in the commented example should also pull from the env-specific credentials

Reply:
Can change it, this was just the default provided by the `ActiveStorage` install script. The thing is, they are probably env vars, not credentials, see discussion below.

Reply:
What about these? Pull them from the env or credentials?